apps/web/test/lib/getSchedule.test.ts

```diff
@@ -1,20 +1,18 @@
 import CalendarManagerMock from "../../../../tests/libs/__mocks__/CalendarManager";
-import prismaMock from "../../../../tests/libs/__mocks__/prisma";
+import prismock from "../../../../tests/libs/__mocks__/prisma";
```

Comment:
What is the case here? Where below we import from prismaMock, should we be concise and rename this as the others? "prismaMock"

Reply:
If we want an In-memory DB implementation we have prismock otherwise we can use our mocked prisma which doesn't implement any method.

So, I believe we should use prismock for integration tests(like booking scenarios) and we can use prismaMock for unit tests(small functions)